app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -148,7 +130,16 @@ class SupplementalApplicationContainer extends React.Component {
     return errors
   }
 
+  checkForValidationErrors = (form) => {
+    const failed = form.getState().invalid
```

Comment:
@chrisdolendo looks like coverage on this PR drops the coverage for the repo. Are you up for adding some more test coverage to the PR? maybe see if you can add a snapshot test for this behavior?

Reply:
Code climate is now happy :)